### ISSUE TITLE
StatusBarItem text overflows outside the status bar

### ISSUE THREAD

**Issue by OmarTawfik** [microsoft/vscode#81456]
- VSCode Version: latest master
- OS Version: darwin 18.7.0

## Steps to Reproduce:

Inside an empty extension, add the following code:
```ts
  const item = vscode.window.createStatusBarItem();
  item.text = 'line 1\nline 2\nline 3';
  item.show();
```

Text should wrap on one line, or be clipped, but instead, this is displayed:

<img width="208" alt="Screen Shot 2019-09-25 at 8 11 42 PM" src="https://user-images.githubusercontent.com/15987992/65655338-b71f8a80-dfd0-11e9-818c-2637b1368849.png">

**Comment by bpasero** [comment]
Could be grid related. @joaomoreno @sbatten

**Comment by joaomoreno** [comment]
No, we can't force the grid to `overflow: hidden` each of its parts since the title menus would break. I've broken that for @sbatten many times already. :laughing: 

Each view is responsible for keeping each of its little piggies within its boundaries.

**Comment by OmarTawfik** [comment]
@bpasero why is this not reproducible? I can still reproduce using the provided example on the latest build.

**Comment by bpasero** [comment]
@OmarTawfik I do not know why it is not reproducible, you have to tell me what else I need to do:

![image](https://user-images.githubusercontent.com/900690/66250589-32a5d800-e745-11e9-8dd0-f3fac3edfc57.png)

**Comment by OmarTawfik** [comment]
I cannot repro on the latest from master now. Thanks for checking!
